ROX-19799: Update docs for VM2.0 (#78059)
openshift/openshift-docs: operating/manage-vulnerabilities/vulnerability-reporting.adoc

:_mod-docs-content-type: ASSEMBLY
[id="vulnerability-reporting"]
= Vulnerability reporting
include::modules/common-attributes.adoc[]
:context: vulnerability-reporting

toc::[]

[role="_abstract"]

> You can create and download an on-demand image vulnerability report from the *Vulnerability Management* -> *Vulnerability Reporting* menu in the {product-title-short} web portal. This report includes a comprehensive list of common vulnerabilities and exposures across images and deployments, called workload CVEs in {product-title-short}. You can share this report with auditors or internal stakeholders by scheduling emails in {product-title-short} or by downloading the report and sharing it by using other methods.

Reviewer: ```suggestion
You can create and download an on-demand image vulnerability report from the *Vulnerability Management* -> *Vulnerability Reporting* menu in the {product-title-short} web portal. This report contains a comprehensive list of common vulnerabilities and exposures in images and deployments, referred to as workload CVEs in {product-title-short}. 
+
To share this report with auditors or internal stakeholders, you can schedule emails in {product-title-short} or download the report and share it by using other methods.
```
